web/themes/custom/pineapple/templates/views/views-view--articles.html.twig

```diff
@@ -62,7 +62,7 @@
   {% endif %}
 
   {% if rows %}
-    <div class="view-content u-o-2">
+    <div class="view-content">
```

Comment:
Put owl class back in and remove margin bottom being added through `_view--articles.scss`